c2corg_ui/templates/helpers/edit.html

```diff
@@ -111,7 +111,19 @@ <h3 class="heading show-phone" translate>summary</h3>
     </div>
   </section>
 </%def>
-
+<%def name="show_cotometer_dialog(module, lang, other_langs)">\
```

Comment:
I see this helper is duplicated in ``c2corg_ui/templates/helpers/view.html``: shouldn't you add it to a common helper file? For instance https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/helpers/common.html

Since it is a route-related helper, it could be added to some route helper file (you may even create one).